pkg/rbd/rbd_util.go

```diff
@@ -269,15 +269,18 @@ func deleteImage(ctx context.Context, pOpts *rbdVolume, cr *util.Credentials) er
 
 	// attempt to use Ceph manager based deletion support if available
 	rbdCephMgrSupported, err := rbdManagerTaskDeleteImage(ctx, pOpts, cr)
-	if !rbdCephMgrSupported {
-		// attempt older style deletion
-		args := []string{"rm", image, "--pool", pOpts.Pool, "--id", cr.ID, "-m", pOpts.Monitors,
-			"--keyfile=" + cr.KeyFile}
-		output, err = execCommand("rbd", args)
+	if rbdCephMgrSupported && err != nil {
+		klog.Errorf(util.Log(ctx, "failed to add task to delete rbd image: %s, %v"), image, err)
+		return err
 	}
 
+	// attempt older style deletion
```

Comment:
This old-style delete should only be done when `rbdCephMgrSupported` is false. Otherwise a 2nd delete is attempted, and that causes the CI to fail.

Reply:
@nixpanic done PTAL